### DISCUSSION TITLE
Add list of mining pools in configuration

### DISCUSSION THREAD

**Discussion by sany3001** [BitMaker-hub/NerdMiner_v2#317] [Ideas]
Recently happen that https://web.public-pool.io/ is not fully working and all my miners are not mining.
It would help to add list of pools into configuration - instead of predefined just one. So change of pool could be more simple. Plus keep that option to provide custom pool.

Reason: list could be created from pools that are usable (at least little verified).
Noobs like me do a little of this topic. And when we want to onboard wide public, we need to think about people who just like how Nerd Miner looks like and do not care how that works.

**Comment by MyOwn2C** [comment]
Maintaining a list of servers that may change in the miner software is not ideal.
If there is a list, every software update will require the coder(s) to check each one in the list to see if they are still active/correct or not.
A google search can easily find a list of BTC solo pools.
Took me 5 seconds to find them.
https://miningpoolstats.stream/bitcoin

**Reply by sany3001** [reply]
What is the difference of having one hard coded value or more hard coded values?

**Reply by MyOwn2C** [reply]
Hard coding is bad programming practice.
I'd rather have server, port and password all blank so users can enter whatever they want.
I think having public pool "pre-filled" is already bad practice. 
Something dynamic like server address should never be hardcoded.

**Comment by BitMaker-hub** [comment]
That is a very cool feature

**Reply by sany3001** [reply]
I agree with @19201003080114 about hard coding in general. 

Ok, so we could solve it by extending manual - adding info that pool address must be entered. But manual is also kind of "hard coded". It is just not FW but SW on GitHub.
